docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -20,6 +20,8 @@ bucket_key = value - rem
 
 From the rounding function above it can be seen that the intervals themselves **must** be integers.
 
+WARNING: Currently, negative values will be incorrectly bucketed so you should make sure all the values will be positive.
```

Comment:
Maybe this should be a bit more specific, like : `WARNING: Currently, values are casted to integers before being bucketed, which might cause negative floating-point values to be in the wrong bucket. For instance, -4.5 with an interval of 2 would be first casted to -4 and would end up in the [-4,-2[ interval instead of [-6, -4[.`